pages/progress.js

```diff
@@ -15,8 +15,27 @@ function HistoryPage() {
     { orderBy: ['timestamp', 'desc'] }
   );
 
-  return fullyLoaded(user, allUserActivities, completedTasks) ? (
-    <History activities={allUserActivities} completedTasks={completedTasks} />
+  const confidentActivityLogEntries = useUserSubcollection('activityLogEntries', {
+    where: ['activityFeeling', 'array-contains', 'Confident'],
+  });
+
+  const allActivityLogEntries = useUserSubcollection('activityLogEntries', {
```

Comment:
Looks like a stale PR -- didn't we get rid of this duplicate?

Reply:
merged in the latest master, duplicates should be gone.